### ISSUE TITLE
Keyboard focus should have better visual feedback

### ISSUE THREAD

**Issue by batzen** [fluentribbon/Fluent.Ribbon#792]
Currently the visual feedback when focusing something with the keyboard is not really good.
This issue tracks the progress towards a better visual feedback.
